code/lib/cli/src/automigrate/fixes/bare-mdx-stories-glob.ts

```diff
@@ -96,29 +98,48 @@ export const bareMdxStoriesGlob: Fix<BareMdxStoriesGlobRunOptions> = {
       .map((entry) => JSON.stringify(entry, null, 2))
       .join('\n');
     return dedent`
-    We've detected your project has one or more globs in your 'stories' config that matches .stories.mdx files:
-      ${chalk.cyan(prettyExistingStoriesEntries)}
-    
-    In Storybook 7, we have deprecated defining stories in MDX files, and consequently have changed the suffix to simply .mdx.
-
-    We can automatically migrate your 'stories' config to include any .mdx file instead of just .stories.mdx.
-    That would result in the following 'stories' config:
-      ${chalk.cyan(prettyNextStoriesEntries)}
-
-    To learn more about this change, see: ${chalk.yellow(
-      'https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mdx-docs-files'
-    )}
-  `;
+      We've detected your project has one or more globs in your 'stories' config that matches .stories.mdx files:
+        ${chalk.cyan(prettyExistingStoriesEntries)}
+      
+      In Storybook 7, we have deprecated defining stories in MDX files, and consequently have changed the suffix to simply .mdx.
+
+      We can automatically migrate your 'stories' config to include any .mdx file instead of just .stories.mdx.
+      That would result in the following 'stories' config:
+        ${chalk.cyan(prettyNextStoriesEntries)}
+
+      To learn more about this change, see: ${chalk.yellow(
+        'https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mdx-docs-files'
+      )}
+    `;
   },
 
-  async run({ dryRun, mainConfigPath, result: { nextStoriesEntries } }) {
+  async run({ dryRun, mainConfigPath, result: { nextStoriesEntries }, packageManager }) {
     logger.info(dedent`✅ Setting 'stories' config:
       ${JSON.stringify(nextStoriesEntries, null, 2)}`);
 
     if (!dryRun) {
+      const glob = await prompt({
+        type: 'text',
+        name: 'glob',
+        message: 'Please enter the glob for your MDX stories',
+        initial: './src/**/*.stories.mdx',
+      });
+
+      await migrate('mdx-to-csf', {
```

Comment:
What do you think about executing `runCodemod` instead of `migrate`? Then you don't have to adjust `migrate` by adding another property `runAutomigration`, which is a bit confusing.